packages/core/src/button/inline-button.element.ts

```diff
@@ -22,11 +22,11 @@ import { styles } from './inline-button.element.css.js';
  * @beta
  * @element cds-inline-button
  * @slot - Content slot for inside the button
+ * @cssprop --text-decoration
  * @cssprop --color
  * @cssprop --font-size
- * @cssprop --font-weight
+ * @cssprop --line-height
```

Comment:
Why the changes here?

Reply:
They didnt match the properties being used in the component https://github.com/vmware/clarity/blob/master/packages/core/src/button/inline-button.element.scss#L9